cst/plugins/eu.esdihumboldt.cst/.classpath

```diff
@@ -3,5 +3,6 @@
 	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER/org.eclipse.jdt.internal.debug.ui.launcher.StandardVMType/JavaSE-1.7"/>
 	<classpathentry kind="con" path="org.eclipse.pde.core.requiredPlugins"/>
 	<classpathentry kind="src" path="src"/>
+	<classpathentry exported="true" kind="lib" path="C:/dev/eclipse/plugins/org.codehaus.groovy_2.3.10.xx-201604081629-e45/lib/groovy-all-2.3.10.jar"/>
```

Comment:
This change should be reverted. It's specific for your system and I don't think you need that anyway to compile the project.